ui/dashboards/src/components/Variables/VariableEditorForm/VariableEditorForm.tsx

```diff
@@ -191,7 +194,7 @@ export function VariableEditForm({
             onChange(getVariableDefinitionFromState(state));
           }}
         >
-          Update Variable
+          Update
```

Comment:
can you switch the update and cancel button to keep it consistent with the rest of the app? i think we have primary button on the left and cancel on the right

Reply:
Yep, will do. I'm also planning to do an overhaul of the UX once all the functionality is complete. We can build re-usable components too to help us be consistent.